content/kotlin/concepts/datetime/terms/toInstant/toInstant.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'toInstant()'
+Description: 'Converts a string representing an instant in ISO-8601 format, which includes both date and time components along with the time zone offset, into an Instant value.'
+Subjects:
+  - 'Mobile Development'
```

Comment:
```suggestion
  - 'Mobile Development'
  - 'Code Foundations'
  - 'Computer Science'
```
You can add all the related subjects, but they need to be listed in the documentation.